docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,328 @@
+---
+
+layout: docs
+
+title:  "Connectable Operators"
+
+section: "chisel3"
+
+---
+
+The `Connectable` operators are the standard way to connect Chisel hardware components to one another.
+
+Note: For descriptions of the semantics for the previous operators, see `connection-operators.md`
+
+All connection operators require the two hardware components (consumer and producer) to be Chisel type-equivalent (matching bundle field names and types, vector sizes, ground types (UInt/SInt)). Use `DataMirror.checkTypeEquivalence` to check this property.
+
+The one exception to the type-equivalence rule is using the `WaivedData` mechansim, detailed at the end of this document.
+
+Chisel types can include data members which are flipped relative to one another. Due to this, there are many desired connection behaviors between two Chisel components. The following are the Chisel connection operators:
+ - `c := p` (mono-direction): assigns all p fields to c; requires c & p to not have flips
+ - `c :#= p` (coercing mono-direction): assigns all p fields to c; ignores all flips
+ - `c :<= p` (aligned-direction); assigns all aligned (non-flipped) c fields from p
+ - `c :>= p` (flipped-direction); assigns all flipped p fields from c
+ - `c :<>= p` (bi-direction operator); assigns all aligned c fields from p; all flipped p fields from c
+
+## Alignment: Flipped vs Aligned
+
+A field's alignment is a relative property; a field is aligned/flipped relative to another member. Hence, one must always say whether a field is flipped/aligned *with respect to (w.r.t)* another member of that type (parent, sibling, child etc.).
```

Comment:
i think you might be well served to give a quick terminology of "component" , "field"  and "member" here and be sure you are only using those terms. For example we say "element" in the code, why are we calling it "member" here? Whats a difference between a "field" and a "member"?